tox.ini

```diff
@@ -15,4 +15,4 @@ commands =
     jwt: pip install -e .[jwt]
 
     flake8 .
-    pytest --cov
+    pytest {posargs:--cov}
```

Comment:
btw why not

```
--cov {posargs}
```

or something similar where we don't disable `--cov` when using positional args?

Reply:
well when trying it locally i don;t usually want coverage to run, if i'm just trying to run tests on a quick cycle

Reply:
Does it meaningfully slow things down? You can also explicitly disable coverage on CLI